src/UserSpaceInstrumentation/Trampoline.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_TRAMPOLINE_H_
+#define USER_SPACE_INSTRUMENTATION_TRAMPOLINE_H_
+
+#include <sys/types.h>
+
+#include <cstdint>
+#include <optional>
+#include <utility>
+#include <vector>
+
+#include "OrbitBase/Result.h"
+
+namespace orbit_user_space_instrumentation {
+
+using AddressRange = std::pair<uint64_t, uint64_t>;
```

Comment:
nit: maybe introduce a trivial class to have start/end instead of first/second, makes code more readable

Reply:
done